test/unit/view/drag-drop-context.spec.js

```diff
@@ -42,6 +42,10 @@ describe('DragDropContext', () => {
 
     const app = TestUtils.findRenderedComponentWithType(tree, App);
 
+    if (!app) {
```

Comment:
This will fail silently. In the places in tests where you are checking for null like this can you please throw an error? Something like this:

```js
if(!app) {
 throw new Error('Invalid test setup');
}